Sources/TuistKit/Generator/ManifestTargetGenerator.swift

```diff
@@ -36,13 +36,8 @@ class ManifestTargetGenerator: ManifestTargetGenerating {
         var buildSettings = [String: String]()
         buildSettings["FRAMEWORK_SEARCH_PATHS"] = frameworkParentDirectory.asString
         buildSettings["LIBRARY_SEARCH_PATHS"] = frameworkParentDirectory.asString
-        buildSettings["SWIFT_FORCE_DYNAMIC_LINK_STDLIB"] = "YES"
-        buildSettings["SWIFT_FORCE_STATIC_LINK_STDLIB"] = "NO"
         buildSettings["SWIFT_INCLUDE_PATHS"] = frameworkParentDirectory.asString
         buildSettings["SWIFT_VERSION"] = Constants.swiftVersion
-        buildSettings["LD"] = "/usr/bin/true"
```

Comment:
I omitted these settings as they are no longer needed to get the target to compile while it's a static framework.